.github/workflows/qualification-report.yaml

```diff
@@ -0,0 +1,49 @@
+on:
+  release:
+    types: [published]
+  pull_request:
+    branches: main
```

Comment:
wasn't sure whether or not to deploy a dev website to get this first "draft" going/see if it actually renders